x-pack/plugins/endpoint/public/applications/endpoint/store/hosts/mock_host_result_list.ts

```diff
@@ -24,13 +24,13 @@ export const mockHostResultList: (options?: {
   // total - numberToSkip is the count of non-skipped ones, but return no more than a pageSize, and no less than 0
   const actualCountToReturn = Math.max(Math.min(total - numberToSkip, requestPageSize), 0);
 
-  const endpoints = [];
+  const hosts = [];
   for (let index = 0; index < actualCountToReturn; index++) {
     const generator = new EndpointDocGenerator('seed');
-    endpoints.push(generator.generateEndpointMetadata());
+    hosts.push(generator.generateHostMetadata());
```

Comment:
this will generate documents from the same host repeatedly with only the timestamp on the document changing.  Including the index in the random seed string will create different documents on each loop iteration.